src/extensions/default/CodeFolding/main.js

```diff
@@ -61,13 +61,14 @@ define(function (require, exports, module) {
     brackets.getModule(["thirdparty/CodeMirror/addon/fold/brace-fold"]);
     brackets.getModule(["thirdparty/CodeMirror/addon/fold/comment-fold"]);
     brackets.getModule(["thirdparty/CodeMirror/addon/fold/markdown-fold"]);
-
+        
```

Comment:
Try to match the indentation.
(Personally I'm not keen on lines with only whitespaces but that is another story)